include/structures/cluster.hpp

```diff
@@ -54,3 +57,12 @@ inline stream_t operator<<(stream_t && stream, Cluster const & clust)
            << clust.get_average_inserted_sequence_size();
     return stream;
 }
+
+/*! \brief A cluster is smaller than another, if its average first mate, average second mate, 
+ *         or average inserted sequence size (in this order) is smaller than the corresponding
+ *         element of the other cluster.
+ *
+ * \param lhs   left side cluster
+ * \param rhs   right side cluster
+ */
+bool operator<(Cluster const & lhs, Cluster const & rhs);
```

Comment:
There is missing a new line at the end of this file. This makes it technical not a text file.
Please add a newline at the end.
Which editor are you using?

Reply:
Well spotted! I'm using VS Code which has the option to insert a new line at the end of each saved file. Unfortunately, it was disabled for me, so I now turned it on.